modules/ROOT/pages/policy-mule4-jwt-validation.adoc

```diff
@@ -23,21 +23,16 @@ endif::[]
 
 |===
 
-== JWT (JSON Web Token)
+JSON Web Token (JWT) is a URL-secure method of representing claims to be transferred between two parties. The claims in a JWT are encoded as a JSON object that is used as the payload of a JSON Web Signature (JWS), or as the plaintext of a JSON Web Encryption (JWE) structure. This enables the claims to be digitally signed and integrity protected with a Message Authentication Code (MAC). Because the token is signed, you can trust the information and its source. 
 
-You can use a JSON Web Token (JWT) to transfer information between parties as a JSON object. Because the token is signed, you can trust the information and its emitter. +
+The JWT Validation policy verifies the signature of the token and asserts the values of the claims of all incoming requests by using a JWT with JWS format. The policy does not validate JWT that uses JSON Web Encryption (JWE).
 
-JSON Web Token (JWT) is a URL-secure method of representing claims to be transferred between two parties. +
-The claims in a JWT are encoded as a JSON object that is used as the payload of a JSON Web Signature (JWS), or as the plaintext of a JSON Web Encryption (JWE) structure. This enables the claims to be digitally signed or integrity protected with a Message Authentication Code (MAC).
+*Eva--are there any prerequisites and scope and considerations to document?*
 
 
-== Policy Overview
+== How This Policy Works
 
-This policy validates incoming requests using a JWT with JWS format. The policy verifies the signature of the token and asserts the values of the claims. This policy does not validate JWT that uses JSON Web Encryption (JWE)
-
-=== Validate Signature
-The policy verifies the signature of the JWT based on the values specified in the policy configuration. +
-The policy rejects all JWTs if the algorithm specified in the policy configuration do not match, or if the signature of the token is invalid. However, if the *None* algorithm option is selected, the policy matches every signed and unsigned token.
+The policy verifies the signature of the JWT based on the values specified in the policy configuration. If the algorithm specified in the policy configuration does not match, or if the signature of the token is invalid all JWTs are rejected by the policy. If no algorithm is specified, the policy matches every signed and unsigned token.
```

Comment:
Policy validates signature and claims. From this paragraph, the second part is not clear

Reply:
@balbifm Please specify what is not clear and what needs to be explained in more detail.

Reply:
This section is like an overview of what the policy does, right? So, what the policy does is to verify the token's signature *and* validate configured claims.
Below this we should probably add a section with title *Signature validation* just like you have *Validate Claims*